src/services/Viewer.js

```diff
@@ -74,6 +75,7 @@ export default class Viewer {
 	 */
 	registerHandler(handler) {
 		this.#state.handlers.push(handler)
+		this.mimetypes.push.apply(this.mimetypes, handler.mimes)
```

Comment:
You still needs a getter. DOn't use the state, it's here just so Vue can make it reactive

Reply:
Haven't found a better way yet :shrug:

Reply:
> You still needs a getter. 

For cleaner API or any technical reason? Because as it is works just fine for me :wink:

Reply:
Cleaner api, but it's getting old, I'm wondering if I can manage reactivity differently :thinking:

Reply:
Pushed.